lib/images.sh

```diff
@@ -6,8 +6,11 @@ if [[ "${IMAGE_OS}" == "Ubuntu" ]]; then
   export IMAGE_NAME=${IMAGE_NAME:-UBUNTU_20.04_NODE_IMAGE_K8S_v1.19.3.qcow2}
   export IMAGE_LOCATION=${IMAGE_LOCATION:-https://artifactory.nordix.org/artifactory/airship/images/k8s_v1.19.3/}
 elif [[ "${IMAGE_OS}" == "FCOS" ]]; then
-  export IMAGE_NAME=${IMAGE_NAME:-fedora-coreos-32.20200923.2.0-openstack.x86_64.qcow2.xz}
-  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/32.20200923.2.0/x86_64/}
+  export IMAGE_NAME=${IMAGE_NAME:-fedora-coreos-32.20201104.3.0-openstack.x86_64.qcow2.xz}
+  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/32.20201104.3.0/x86_64/}
```

Comment:
Are we missing something in the image path ?

```
$ wget https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/32.20201104.3.0/x86_64/fedora-coreos-32.20201104.3.0-openstack.x86_64.qcow2.xz

--2020-12-20 13:14:27--  https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/32.20201104.3.0/x86_64/fedora-coreos-32.20201104.3.0-openstack.x86_64.qcow2.xz
Resolving builds.coreos.fedoraproject.org (builds.coreos.fedoraproject.org)... 205.251.219.79, 205.251.219.37, 205.251.219.127, ...
Connecting to builds.coreos.fedoraproject.org (builds.coreos.fedoraproject.org)|205.251.219.79|:443... connected.
HTTP request sent, awaiting response... 403 Forbidden
2020-12-20 13:14:28 ERROR 403: Forbidden.

```

Reply:
Ah thanks, I think the update to the openstack image was a mistake, now reverted to the previous version and we can perhaps update it in a future commit.